src/com/google/javascript/jscomp/PolymerClassDefinition.java

```diff
@@ -33,6 +33,8 @@
  * class.
  */
 final class PolymerClassDefinition {
+  /** Either 1 or 2 - the Polymer version of the definition */
+  final int version;
```

Comment:
That would definitely make things clearer. We could change the flag to be `--polymer_pass=VERSION`.